### DISCUSSION TITLE
Skip Chain Selection in AppKit Modal for Preselected Network in Multichain Wallet

### DISCUSSION THREAD

**Discussion by andremfaria** [reown-com/appkit#2972] [Q&A]
Hello,

I'm working on a dApp where the user selects the desired chain outside the AppKit context before opening the AppKit modal.

Currently, when the user clicks on the wallet labeled as "MULTICHAIN" (1), they are prompted to select the chain (Ethereum or Solana) (2).

Is there a way to bypass step (2) and directly connect to the preselected chain?

Version :  "@reown/appkit": "1.0.4"
Thanks.

(1)
<img width="349" alt="Screenshot 2024-10-01 at 14 07 22" src="https://github.com/user-attachments/assets/81c081d7-3fe4-40b7-8765-69eca5f5fe76">

(2)

<img width="326" alt="Screenshot 2024-10-01 at 14 07 32" src="https://github.com/user-attachments/assets/b641f374-e41c-42da-86ba-504b14f3dba9">

**Comment by marcobusico** [comment]
Have you found any solution to this issue?

**Reply by rtomas** [reply]
yes, we are working on this. Will be released in one or two weeks.

**Reply by jimmy-xyz** [reply]
Is there any progress?
